packages/react/src/hooks/useSignIn.ts

```diff
@@ -24,6 +26,10 @@ export const useSignIn: UseSignIn = () => {
   const isomorphicClerk = useIsomorphicClerkContext();
   const client = useClientContext();
 
+  useEffect(() => {
+    isomorphicClerk.telemetry?.record(eventMethodCalled('useSignIn'));
```

Comment:
Hooks aren't necessary methods per se. However, I don't see the benefits of creating another event type for it, cause it wouldn't change much in terms of querying that data. 

Anyway, open for feedback here!